src/main/res/drawable/ic_settings_upload_photo.xml

```diff
@@ -4,6 +4,6 @@
     android:viewportWidth="24"
     android:viewportHeight="24">
     <path
-        android:fillColor="#000000"
+        android:fillColor="@color/accent"
```

Comment:
Hmm, such references don't work for Android with SDK version less than 21. Since we recently changed the `minSDKVersion` to 21, this shouldn't be an issue. However, keeping `android:tint` overrides this drawable property.